_projects/2024/100542532/100542532.Rmd

```diff
@@ -0,0 +1,532 @@
+---
+title: "Exploring Where Americans Can Afford to Live Alone"
+description: | 
+  Rising rents and stagnant wages make even modest apartments unaffordable in many U.S. cities
+categories: "2024"
+author: Isabel Monge
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+This double lollipop graph below, [originally published by The Economist in 2023](https://www.economist.com/graphic-detail/2023/08/04/our-carrie-bradshaw-index-where-americans-can-afford-to-live-solo), compares the median wage to the affordable studio wage across cities in the United States. It highlights the growing housing affordability crisis and reveals where single renters face the greatest financial challenges. My objective is to recreate this graph as closely as possible using R, and then to propose an alternative visualization.
+
+```{r, echo=FALSE, out.width='100%'}
+knitr::include_graphics("Economist_Graph_SS.png")
+```
+
+## Replication
+
+```{r}
+#| message: false
+
+library(ggplot2)   # Create plots
+library(showtext)  # Custom fonts
+library(sysfonts)  # Manage fonts
+library(readr)     # Read data
+library(ggiraph)   # Add tooltips
+library(shiny)     # Build interactive web applications (used for dropdown list)
+library(rvest)     # Web scraping and extracting data from HTML
+library(dplyr)     # General data manipulation
+library(stringr)   # String manipulation and cleaning
+library(maps)      # Plot basic map outlines
+library(sf)        # Handle spatial data (maps)
+library(leaflet)   # Create interactive maps
+library(scales)    # Format tooltips with commas for better readability
+```
+
+### **Data**
```

Comment:
No need to set it as bold. Please any additional formatting like this from all the titles, so that the style is uniform with respect to other posts in the web.